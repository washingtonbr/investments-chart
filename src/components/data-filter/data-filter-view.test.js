import React from 'react';
import renderer from 'react-test-renderer';
import DataFilterView from './data-filter-view';

describe('DataFilterView', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<DataFilterView />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
