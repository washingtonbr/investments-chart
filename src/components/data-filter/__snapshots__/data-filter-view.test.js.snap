// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataFilterView renders correctly 1`] = `
<div
  className="sc-AxjAm iOWkxF"
>
  <select
    className="sc-AxirZ gdDDhx"
  >
    <option
      className="sc-AxiKw kgEfKZ"
      selected={true}
    >
      Desde o início
    </option>
    <option
      className="sc-AxiKw kgEfKZ"
    >
      Último mês
    </option>
    <option
      className="sc-AxiKw kgEfKZ"
    >
      3 meses
    </option>
    <option
      className="sc-AxiKw kgEfKZ"
    >
      1 ano
    </option>
    <option
      className="sc-AxiKw kgEfKZ"
    >
      2 anos
    </option>
  </select>
</div>
`;
